package.json

```diff
@@ -8,9 +8,11 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "classnames": "^2.2.6",
     "prop-types": "^15.7.2",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-intersection-observer": "^8.31.0",
```

Comment:
I am using this in `ChipList` to detect whether a chip is in view or not

Reply:
Haven't used it, but looks well-maintained.